salesagility/SuiteCRM
# Map add to targetlist - json reply

in maps after clicking add to target list label was stuck in "Processing ...".

## Description
json_encode reutrned error 6.
Simplified object for json, and now Proccessing chages to target list name...

## Motivation and Context
Label on Maps stucked in Processing after clicking on Add to targetlist...

## How To Test This
Go to Map module/display map. Click Add to target List. Label should change to Processing and after few seconds to name of the target list ...

## Types of changes
- [x ] Bug fix (non-breaking change which fixes an issue)

### Final checklist
